packages/service/vivo.go

```diff
@@ -19,44 +20,41 @@ func notFound(w http.ResponseWriter) {
 	io.WriteString(w, "404 page not found")
 }
 
-func VivoListen(frontendStaticDir string) {
-	fs := http.FileServer(http.Dir(frontendStaticDir))
-	http.Handle("/", fs)
-
-	http.HandleFunc("/vivo/", func(w http.ResponseWriter, req *http.Request) {
-		if req.Method != "GET" {
-			notFound(w)
-			return
-		}
+func vivoForward(w http.ResponseWriter, req *http.Request) {
+	if req.Method != "GET" {
+		notFound(w)
+		return
+	}
 
-		switch req.URL.Path {
-		case "/vivo/logs":
-		case "/vivo/metrics":
-		case "/vivo/traces":
-		default:
-			notFound(w)
-			return
-		}
+	client := &http.Client{}
+	//http: Request.RequestURI can't be set in client requests.
+	//http://golang.org/src/pkg/net/http/client.go
+	req.RequestURI = ""
+	req.Host = "127.0.0.1"
+	req.URL.Host = "127.0.0.1:2025"
```

Comment:
Do we need the port? If so, maybe as an env var in the container?

Reply:
Well, both could use an env var... - and the config can be changed at runtime but this cannot if hardcoded.
My concern is someone changes one but the other is hardcoded.

Reply:
Are you saying it is possible to change fluent-bit vivo exporter port at runtime? I was under the impression everything was defined at build time by the docker image.

I don't see how changing it at runtime can work, wouldn't it break kubernetes port mapping? The ports are also hardcoded in kubernetes yaml files.

Reply:
This is just an image, yes we have a deployment but anyone can make their own - there is no coupling between the image and the K8S configuration. I just wanted to support the option of people using other ports but it is not a requirement so no worries.

We can expose whatever ports we like from a container at runtime, the `EXPOSE` keyword is just to list expected ones - there is no restriction it can only expose those ones or even that it will expose those ones: https://docs.docker.com/engine/reference/builder/#expose
In some situations it will auto-map ports and also allow inter-contain communication on the same host/network if they are exposed but ultimately it is up to the container runtime what ports are open.